srcpkgs/python3-pylast/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'python3-pylast'
 pkgname=python3-pylast
-version=4.1.0
-revision=5
-build_style=python3-module
-hostmakedepends="python3-setuptools_scm"
-depends="python3-setuptools"
+version=5.1.0
+revision=1
+build_style=python3-pep517
+hostmakedepends="hatchling hatch-vcs"
+depends="python3-httpx"
 short_desc="Python3 interface to last.fm and libre.fm"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="Apache-2.0"
 homepage="https://github.com/pylast/pylast"
 distfiles="${PYPI_SITE}/p/pylast/pylast-${version}.tar.gz"
-checksum=ad084aec1bf7e307bc42d7cc1a003851f5bee1ad24fb697a9fdc300bbfe63932
-make_check=no # The 'pylast' distribution was not found and is required by the application
+checksum=89300fdcdf423d7be0606bdc44da27c3b48c4d73aa1d4cb12672cc006c979bdc
+make_check=no # Unable to determine tests for PEP517 Python templates
```

Comment:
that's a harmless warning, there should be no reason to explicitly disable tests, also, we can run at least some tests, it's currently ugly, but it works.
```
checkdepends="python3-pytest python3-flaky ${depends}"
do_check() {
       rm -rf tmp # remove artifacts from previous unsuccessful check run
       mkdir tmp
       bsdtar xf dist/pylast-$version-py3-none-any.whl -C tmp
       PYTHONPATH=tmp pytest
       rm -rf tmp
}
```

Reply:
now when https://github.com/void-linux/void-packages/pull/42013 has been merged, you can leave out the do_check function